src/main/scala/com/gu/contentapi/firehose/client/PublicationLogic.scala

```diff
@@ -1,6 +1,7 @@
 package com.gu.contentapi.firehose.client
 
 import com.gu.contentapi.client.model.v1.Content
+import com.gu.crier.model.event.v1.EventPayload.Atom
```

Comment:
If you change this import to `import com.gu.contentatom.thrift.Atom` then you won't need to cast in `ContentApiEventProcessor`

Reply:
great, thanks.  sometime it would be good to have a chat about what that AtomAlias actually is.